engines/python/setup/djl_python/rolling_batch/trtllm_rolling_batch.py

```diff
@@ -24,9 +24,6 @@ def __init__(self, model_id_or_path, device, properties, **kwargs):
         :param properties: other properties of the model, such as decoder strategy
         """
         super().__init__(-1, **kwargs)
-        if properties.get("engine") != "MPI":
```

Comment:
We can check `mpi_mode` instead of check engine.

We just need to manually set mpi_model in `PyModel`
```
properties.put("mpi_mode", "true");
``